cmd/show.go

```diff
@@ -60,13 +60,19 @@ ks show prod -c redis -o json
 ks show dev -c redis -c nginx-server
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 1 {
-			return fmt.Errorf("'show' takes at most a single argument, that is the name of the environment")
+		if len(args) != 1 {
+			return fmt.Errorf("'show' requires an environment name; use `env list` to see available environments")
 		}
+		env := args[0]
```

Comment:
There is a chance that `args` could be `0` since you only checking for `!= 1`.  This could panic.